components/router/demo/demo.js

```diff
@@ -20,8 +20,9 @@ import {
     notFoundTemplate,
 } from './templates';
 
-const browserHistory = new BrowserHistory();
-Route.use(browserHistory);
+// window.cohHistory  = new BrowserHistory();
+window.cohHistory = new HashHistory();
```

Comment:
why don't we create a demo with BrowserHistory and a demo with HashHistory? Don't want to have them both?

Reply:
I do this in `demo-browser-history.html` and `demo-hash-history.html`